src/Auth/Passport/AuthStrategy.ts

```diff
@@ -47,51 +38,15 @@ export class AuthStrategy {
     })(req, res, next)
   }
 
-  public static scopedJWTAuth(scopeName: string) {
-    return (req: Request, res: Response, next: NextFunction) => {
-      const scope = config.scopes.find((s) => s.name === scopeName)
-      if (!scope) {
-        throw new ValidationError('scope not found', scopeName)
-      }
-      ScopedJwtAuthStrategy.use({ scope: scope })
-      passport.authenticate(AuthStrategyTypes.scopedJwt, (error: Error, user: any) => {
-        if (user && !error) {
-          req.user = user
-          return next()
-        }
-        return next(new InvalidCredentialsError('Invalid token.'))
-      })(req, res, next)
-    }
-  }
-
   /**
    * Returns anExpress Middleware function that reads application ID and application secret values from headers or query parameters, and validates them
    */
-  public static applicationValidation() {
+  public static secretValidation() {
     return async (req: Request, _res: Response, next: NextFunction): Promise<void> => {
-      const applicationRepository = DIContainer.sharedContainer.applicationRepository
-
-      let appId = req.headers[APP_ID_HEADER_KEY]
-      let appSecret = req.headers[APP_SECRET_HEADER_KEY]
-
-      if (!appId) {
-        appId = req.query[APP_ID_HEADER_KEY] as string
-        appSecret = req.query[APP_SECRET_HEADER_KEY] as string
+      const appSecret = req.headers[APP_SECRET_HEADER_KEY]
```

Comment:
@mnatsheh I kept the name of the header as is "manuscripts-app-secret", since it isn't affecting what it stands for, any concerns?